cli/tests/end_to_end/extension.rs

```diff
@@ -0,0 +1,128 @@
+use std::fs;
+use std::path::Path;
+
+use assert_cmd::assert::Assert;
+use assert_cmd::Command;
+use tempfile::TempDir;
+
+use crate::end_to_end;
+
+const HEADER: &str = "import { PhylumApi } from 'phylum';";
+
+/// Test Phylum API functions.
+#[tokio::test]
+pub async fn api() {
```

Comment:
I'd use more verbose function names for tests, e.g. api_called_correctly_from_extensions or something along these lines. I've started doing that recently and I feel it makes things a lot clearer, whereas I had to read the body to understand what api meant at first.

Reply:
I think the key is to avoid redundancy. We're already in the extensions module so this test is already `extensions::api`, making it `extensions::api_called_correctly_from_extensions` doesn't provide much additional benefit, of course you're checking that it works correctly because that's the default you'd expect for any test that doesn't point out otherwise, you know it's an extension because that's the module we're in and already part of the test name.

So I don't mind making the function names more explicit, but it should provide additional information that doesn't exist yet, rather than just making it harder to decipher the concrete purpose of the test from its name.

Reply:
That specific one was a random suggestion. The core concept is that `extensions::api` is also not immediately clear to me and anything that makes this clearer would be better, really.